### DISCUSSION TITLE
Audio Balance shifted completely towards the right

### DISCUSSION THREAD

**Discussion by arkojeet-bera** [ExistentialAudio/BlackHole#490] [Q&A]
Hello👋
I'm running macOS Monterey and I've run into a slight issue. I've installed BlackHole yesterday and during the installation suddenly the **audio balance of my Bluetooth earphone shifted completely towards the right**. Then I corrected it from the sound preferences but seems like it has done something to it. Every time I reconnect my Bluetooth earphone, the sound balance is always shifted towards the right and I've to manually correct it. I've uninstalled Blackhole from the uninstaller and also ran the command `sudo launchctl kickstart -kp system/com.apple.audio.coreaudiod` to restart core audio, but still, the issue persists.
Please help🙏
For reference, I'm attaching a Video to demonstrate the issue properly:
[https://drive.google.com/file/d/1tHZ2LJfEFi3KH2vs0jc45BJTDOeJa9lB/view?usp=sharing](url)

**Comment by devinroth** [comment]
That’s a strange one. You’ve uninstalled BlackHole so it’s clearly not causing the issue.

Have you restarted? How about remove and and pair your headphones again?

Devin

**Reply by arkojeet-bera** [reply]
Yes, I’ve done both, but still, it persists.

**Comment by devinroth** [comment]
Maybe a hard reset on the headphones. Try contacting the manufacturer.

Devin
